backend/fixtures/postgres/sql/job_offers.sql

```diff
@@ -1,10 +1,20 @@
-insert into job_offers (id, email, company_name, title_id, is_remote, description, how_to_apply, apply_url, apply_email_address, createdat, updatedat) values (8, 'bnorker7@mit.edu', 'Camido', 8, true, 'Aenean fermentum. Donec ut mauris eget massa tempor convallis. Nulla neque libero, convallis eget, eleifend luctus, ultricies eu, nibh. Quisque id justo sit amet sapien dignissim vestibulum. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nulla dapibus dolor vel est. Donec odio justo, sollicitudin ut, suscipit a, feugiat et, eros. Vestibulum ac est lacinia nisi venenatis tristique. Fusce congue, diam id ornare imperdiet, sapien urna pretium nisl, ut volutpat sapien arcu sed augue.', 'Suspendisse accumsan tortor quis turpis. Sed ante. Vivamus tortor.', 'https://cdc.gov/varius/integer/ac/leo/pellentesque/ultrices.html', 'mruppertz7@ed.gov', '8/20/2022', '12/13/2022');
-insert into job_offers (id, email, company_name, title_id, is_remote, description, how_to_apply, apply_url, apply_email_address, createdat, updatedat) values (9, 'kbattill8@chron.com', 'Camimbo', 9, true, 'Duis at velit eu est congue elementum. In hac habitasse platea dictumst. Morbi vestibulum, velit id pretium iaculis, diam erat fermentum justo, nec condimentum neque sapien placerat ante. Nulla justo. Aliquam quis turpis eget elit sodales scelerisque. Mauris sit amet eros. Suspendisse accumsan tortor quis turpis. Sed ante. Vivamus tortor. Duis mattis egestas metus.', 'Nulla ac enim. In tempor, turpis nec euismod scelerisque, quam turpis adipiscing lorem, vitae mattis nibh ligula nec sem.', 'http://reuters.com/commodo/placerat/praesent/blandit/nam/nulla/integer.png', 'ggarrigan8@nbcnews.com', '8/28/2022', '2/14/2023');
-insert into job_offers (id, email, company_name, title_id, is_remote, description, how_to_apply, apply_url, apply_email_address, createdat, updatedat) values (10, 'cbinfield9@github.com', 'Zoombox', 10, false, 'In hac habitasse platea dictumst. Maecenas ut massa quis augue luctus tincidunt. Nulla mollis molestie lorem. Quisque ut erat. Curabitur gravida nisi at nibh. In hac habitasse platea dictumst. Aliquam augue quam, sollicitudin vitae, consectetuer eget, rutrum at, lorem. Integer tincidunt ante vel ipsum. Praesent blandit lacinia erat.', 'Nulla tellus. In sagittis dui vel nisl. Duis ac nibh.', 'http://salon.com/quam/nec.html', 'sbenoit9@example.com', '8/20/2022', '5/13/2022');
+insert into job_offers (id, email, company_name, title_id, is_remote, description, how_to_apply, apply_url, apply_email_address, createdat, updatedat) 
```

Comment:
Same remark here, this can be formated, upperCased and transform as one query for a bulk insert.